qiskit_nature/second_q/mappers/linear_mapper.py

```diff
@@ -86,7 +73,7 @@ def _linear_encoding(self, spin: Union[Fraction, float]) -> List[PauliSumOp]:
             to represent the embedded 'X' operator
         """
 
-        spin_op_encoding: List[PauliSumOp] = []
+        spin_op_encoding: list[PauliSumOp] = []
```

Comment:
I wonder whether instead of appending to this list we could just construct it once in the end, the same way to do it in the `LogarithmicMapper`